File: backend/src/Modules/Modules.Content.Application/Articles/GetPublishedBySlug/PublishedArticleDetails.cs
namespace Modules.Content.Application.Articles.GetPublishedBySlug;

public sealed record PublishedArticleDetails(
    Guid Id,
    string Slug,
    string TitleHu,
    string? TitleEn,
    string? SummaryHu,
    string? SummaryEn,
    string ContentHu,
    string? ContentEn);

File: frontend/henn-laszlo/projects/content-data-access/src/lib/models/published-article-details.ts
export interface PublishedArticleDetails {
  readonly id: string;
  readonly slug: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly summaryHu: string | null;
  readonly summaryEn: string | null;
  readonly contentHu: string;
  readonly contentEn: string | null;
}